src/ProjectOrigin.Registry.IntegrationTests/PerformanceTests.cs

```diff
@@ -72,6 +77,15 @@ public class PerformanceTests : IAsyncLifetime, IClassFixture<ContainerImageFixt
                 .WithEnvironment("Logging__LogLevel__Grpc.AspNetCore", "Information")
                 .WithEnvironment("Cache__Type", "redis")
                 .WithEnvironment("Cache__Redis__ConnectionString", redisFixture.ContainerConnectionString)
+                .WithEnvironment("RabbitMq__Hostname", rabbitMqFixture.ContainerIp)
+                .WithEnvironment("RabbitMq__AmqpPort", RabbitMqFixture.ContainerAmqpPort.ToString())
+                .WithEnvironment("RabbitMq__HttpApiPort", RabbitMqFixture.ContainerHttpPort.ToString())
```

Comment:
The port is dynamic correct?

Reply:
could be redirected in kubernetes services, so therefore it is possibly to configure them.